pkg/controller/jobframework/reconciler.go

```diff
@@ -310,6 +310,19 @@ func (r *JobReconciler) ReconcileGenericJob(ctx context.Context, req ctrl.Reques
 		return ctrl.Result{}, client.IgnoreNotFound(err)
 	}
 
+	if features.Enabled(features.ManagedJobsNamespaceSelector) &&
```

Comment:
@mimowo @tenzen-y 

What if we graduate `features.ManagedJobsNamespaceSelector` to GA?